tests/test_cli.py

```diff
@@ -65,6 +66,7 @@ def setUpClass(cls):
                 u'- お早う御座います。\n'
                 u'# الأَبْجَدِيَّة العَرَبِيَّة\n').encode('utf-8'),
         })
+        cls.conf = config.YamlLintConfig('extends: default')
```

Comment:
Can you remove this and just create `conf` in `test_find_files_recursively()`?

Also, can you add other tests in `test_find_files_recursively()`, like running with:
- ```python
  conf = config.YamlLintConfig('extends: default\n'
                               'yaml-files:\n'
                               '  - *.yaml\n')
  ```
- ```python
  conf = config.YamlLintConfig('extends: default\n'
                               'yaml-files:\n'
                               '  - *.yml\n')
  ```
- ```python
  conf = config.YamlLintConfig('extends: default\n'
                               'yaml-files:\n'
                               '  - *.json\n')
  ```
- ```python
  conf = config.YamlLintConfig('extends: default\n'
                               'yaml-files:\n'
                               '  - *\n')
  ```
- ```python
  conf = config.YamlLintConfig('extends: default\n'
                               'yaml-files:\n'
                               '  - *.yaml\n'
                               '  - *\n'
                               '  - **\n')
  ```
- ```python
  conf = config.YamlLintConfig('extends: default\n'
                               'yaml-files:\n'
                               '  - s/**\n'
                               '  - **/utf-8\n')
  ```

Reply:
partially done, applied remark with moving `conf` object, will try to find time for adding tests later today

Reply:
@adrienverge added requested tests